packages/pluggableWidgets/accordion-web/src/Accordion.tsx

```diff
@@ -5,19 +5,42 @@ import { Accordion as AccordionComponent, AccordionGroups } from "./components/A
 import { useIconGenerator } from "./utils/iconGenerator";
 
 import { AccordionContainerProps } from "../typings/AccordionProps";
+import { Header } from "./components/Header";
 
 export function Accordion(props: AccordionContainerProps): ReactElement | null {
     const accordionGroups: AccordionGroups | undefined = useMemo(() => {
-        if (props.groups.some(group => group.visible.value === undefined || group.headerText.value === undefined)) {
+        if (
+            props.groups.some(
+                group =>
+                    group.visible.value === undefined ||
+                    group.headerText.value === undefined ||
+                    group.initiallyCollapsed.value === undefined ||
+                    (group.collapsed && group.collapsed.value === undefined)
```

Comment:
Nit: This is becoming quite big now so it's hard to quickly determine what this does. Maybe it's nice to extract into separate function?

Reply:
Changed it.